scripts/qa_crowbarsetup.sh

```diff
@@ -2496,6 +2496,13 @@ function custom_configuration
             if [[ $want_keystone_v3 ]] ; then
                 proposal_set_value keystone default "['attributes']['keystone']['api']['version']" "'3'"
             fi
+            if iscloudver 7plus && [[ $want_keystone_token_type ]]; then
+                case "$want_keystone_token_type" in
+                    fernet|uuid)
```

Comment:
I've added `case` condition as validator, value will be configured will be set only for "fernet" or "uuid" value